modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -488,9 +488,8 @@ sub _load_in_species_pages {
   return $spp_tree;
 }
 
-sub _load_in_taxonomy_division {
-  my ($self) = @_;
-  my $filename = $SiteDefs::ENSEMBL_TAXONOMY_DIVISION_FILE;
+sub _load_json_config {
+  my ($self, $filename) = @_;
   my $json_text = do {
```

Comment:
Make this method more generic, so it can be used to parse other JSON config files